products/centos.md

```diff
@@ -14,21 +14,6 @@ identifiers:
 -   cpe: cpe:2.3:o:centos:centos
 
 releases:
--   releaseCycle: "9"
-    releaseDate: 2021-09-15
-    support: 2027-05-31
-    eol: 2027-05-31
-    latest: "9"
-    link: https://blog.centos.org/2021/12/introducing-centos-stream-9/
-
--   releaseCycle: "stream-8"
-    releaseLabel: "CentOS Stream 8"
-    releaseDate: 2019-09-24
-    support: 2024-05-31
-    eol: 2024-05-31
-    latest: "8"
-    link: https://wiki.centos.org/Manuals/ReleaseNotes/CentOSStream
-
 -   releaseCycle: "8"
     releaseLabel: "CentOS Linux 8"
```

Comment:
```suggestion
```



Now that CentOS and CentOS Stream are not mixed on the same page this does not seem necessary anymore.

Reply:
We may want to apply the same update to 6 and 7.